src/radical/entk/execman/rp/task_processor.py

```diff
@@ -472,7 +491,7 @@ def create_cud_from_task(task, placeholders, prof=None):
             prof.prof('cud from task - done', uid=task.uid)
 
         logger.debug('CU %s created from Task %s' % (cud.name, task.uid))
-
+        
```

Comment:
Your editor introduces trailing spaces it seems.  I don't mind, but the linter does...

Reply:
I'll fix this

Reply:
FIxed